lib/functions-custom.php

```diff
@@ -1,3 +1,39 @@
 <?php
 
-// Custom functions (like special queries, etc)
\ No newline at end of file
+// Custom functions (like special queries, etc)
+
+function get_events_array( $query_args = '' ) {
+  $args = wp_parse_args( $query_args, array(
+    'post_type' => 'evento',
+    'meta_key' => '_igv_evento_start',
+    'order' => 'ASC',
+    'orderby' => 'meta_value',
+    'meta_query' => array(
+      array(
+        'key'     => '_igv_evento_start',
+        'value'   => current_time('timestamp') - (60 * 60 * 24), // UTC -5, 6 hours ago
```

Comment:
moment.php is built in to our boiler and will generate these time values with a much more readable codebase. There is a function `subtractHours()` on moment objects that will do this manipulation readably

However why does the function subtract 6 hours? So there is stuff that previously happened, or is this related to servertime?